test/cracking_test.rb

```diff
@@ -32,10 +33,15 @@ def test_it_can_crack_the_message_with_a_date
     expected = { decryption: 'hello world end', date: '291018', key: '08304' }
     assert_equal expected, crack.crack
 
-    encryptor = Encryptor.new('message is this end')
+    encryptor = Encryptor.new('message is that it is hard to decrypt end', '08304')
     output = encryptor.encrypt 
     crack = Cracking.new(output[:encryption])
-    expected = { decryption: 'message is this end', date: Time.now.strftime('%d%m%y'), key: output[:key] }
+    crack.stubs(:find_key_from_date).returns('08304')
```

Comment:
stubs out the `find_key_from_date` method (tested a few lines above). This considerably reduces overall testing time